src/StatsdClient/StatsdClient.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>DogStatsD-CSharp-Client</PackageId>
     <Description>A DogStatsD client for C#. DogStatsD is an extension of the StatsD metric server for use with Datadog. For more information visit http://datadoghq.com.</Description>
     <Authors>Datadog</Authors>
-    <TargetFrameworks>netstandard1.3;netstandard1.6;netcoreapp1.1;netcoreapp2.0;net451;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard1.6;netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
```

Comment:
Can we revert this small change?

1) dropping `net451` is a breaking change (users on .NET Framework 4.5.1 will no longer be supported)
2) [official guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting) is to include `net461` when offering `netstandard2.0`:

>✔️ CONSIDER adding a target for `net461` when you're offering a `netstandard2.0` target.
>
>Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.

Reply:
Sadly net451 doesn't have the `UDPSocket.SendToAsync` method which i'm using in the code.

Is it really necessary to support net45 when I believe it is no longer supported by microsoft?